packages/terra-menu/src/MenuItem.jsx

```diff
@@ -294,11 +294,25 @@ class MenuItem extends React.Component {
     ]);
 
     let content = textContainer;
+    let navigationContent;
+    let ariaLiveValue = 'polite';
+
+    if (this.itemNode && index === 0 && totalItems !== 1) {
+      navigationContent = intl.formatMessage({ id: 'Terra.menu.navigateMenuItem' });
+      if (!MenuUtils.isMac() && this.itemNode.parentNode.getAttribute('data-submenu') === 'false') {
```

Comment:
getAttribute returns null when it does not find the specified element so the above condition will not work as you are checking for 'false'

```suggestion
      if (!MenuUtils.isMac() && !this.itemNode.parentNode.getAttribute('data-submenu')) {
```

Reply:
Changed sub menu check condition - [2680850](https://github.com/cerner/terra-framework/pull/1718/commits/268085011ef85a139212575de2028c3f128d4265)